src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
One of the goals of CRAFT, maybe. Not my goal in particular. I just want to build cool hardware. 

But I agree. In the perfect world, if everyone cared about reusability by other people (and not one-off ISSCC papers), then yes, more attention should be paid to infrastructure. It's just that deadlines are really out of our (or at least *my*) control. If people who had more control over these deadlines understood that people needed to nominally spend ~1 year on infrastructure before tapeout (and that it's kind of fruitless to tapeout while infrastructure is in its infancy), then we'd have a much more reasonable experience. Future tapeouts would involve less wastes of time rebuilding the same infrastructure over and over again. [Heck, that can be said about pretty much all of BWRC's "house of cards" infrastructure]. In the end, though, from many years of observation, a PhD is perceived(maybe wrongly so?) to be something *you* did in whatever field you're supposed to be in, so trying to be open source-y altruistic on something that's a tool for succeeding in your specialty unfortunately becomes somewhat tangential to that -- which kind of leads to this messy infrastructure -- but if no one else before you bothered cleaning up the mess, then is it really in *your* best interest to spend more time making things better for the people after you instead of getting papers in your name, etc. so you can graduate? It's a different kind of merit that makes more sense at a company or elsewhere (or maybe if your PhD specializes in operations management or something), where team success is your success (and keeps getting you a paycheck). But up until now, I haven't really seen this philosophy working in a PhD setting... [If that actually worked, my super tapeout wouldn't have been such a nightmare...]

For things that are easy to build upon, I'm totally fine helping out, but when I'm choosing between my own deliverables and the "general good" within a constrained time frame, it only makes sense that I focus on the former. I have a *bleep* ton of ideas for things I'd like to see in Chisel to be useful for *circuit designers* in the future; some of which I do plan on working on... but I don't want to fight the system to make it work. That's why I prefer layering on top of Chisel things via a library (which is wholly up to me or someone else at a higher-than-Chisel level to maintain), but it's just not easy to do if you don't have access to everything in Chisel. If I didn't care about infrastructure at all, I'd just be building all of my functionality into a "for Angie only repo", but that's not exactly how I've been working for the last year; otherwise, I probably would've been done  a lot sooner -- in that case, why care at all about user experience, right? Since I'm the only user. If nothing else, though, hacking a solution to a problem that hasn't been previously addressed by Chisel is still useful in the sense that at least people become aware that x, y, z feature can actually be useful. Then Chisel retainers or w/e can decide if it's useful enough to solidify and build into the core.

Anyways, all this talk about infrastructure and whatnot makes me think we should bring in some IEOR PhD student to figure out how to coordinate or get more industry people involved.